Source/OxyPlot/Annotations/ArrowAnnotation.cs

```diff
@@ -183,6 +183,11 @@ public override void Render(IRenderContext rc)
         /// </returns>
         protected override HitTestResult HitTestOverride(HitTestArguments args)
         {
+            if (!this.PlotModel.PlotArea.Contains(args.Point))
```

Comment:
move this into calling method?

Reply:
@VisualMelon Please, can you tell me how can I access the `(PlotModel.)PlotArea` from the calling method of the code above, [`UIElement.HitTest`](https://github.com/silviubogan/oxyplot/blob/5bc747c1eda59d55953ac70bf1b8a37af87bb29f/Source/OxyPlot/Graphics/UIElement.cs#L69)? Should there be a new `UIElement.PlotModel` property? Thank you.

Reply:
I'm not sure. `UIElement.PlotModel` would couple `UIElement` to `PlotModel` unnecessarily. I'm not sure if objorke means that the decision to test agains the PlotArea should be handled higher up (though currently it cannot, because `Model` and not `PlotModel` do the work), or if the idea is just to reduce redundancy.

I am not totally convinced that the elements should be filtered at this layer, and I wonder if `Model.HitTest` should give more control to the inheriting class. That's all assuming the change is desirable, which I don't think is entirely clear with annotations.